Samples/Samples/Samples.csproj

```diff
@@ -24,4 +24,10 @@
     <ProjectReference Include="..\..\Xamarin.Essentials\Xamarin.Essentials.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="View\ColorConvertersPage.xaml.cs">
```

Comment:
I see VS is up to its old tricks again :)

Reply:
Good catch! Fixed :)